decision-trees.Rmd

```diff
@@ -300,6 +298,8 @@ By comparison, let's look at the tree for the third fold.
 rpart.plot(all_mods[[3]], type = 4)
 ```
 
+<!-- BC: The trees for fold 1 and fold 3 actually look almost identical, so the statement below is confusing. Perhaps something changed since you first wrote this? -->
+
```

Comment:
@datalorax For some more clarification here -- these are the trees I'm seeing when previewing this chapter locally. They look almost (but not quite) identical. But it seems like you were originally seeing these as quite different? 

### Fold 1 
![Screen Shot 2020-12-30 at 1 01 06 PM](https://user-images.githubusercontent.com/32279312/103381327-4c48e300-4aa0-11eb-8b50-700c41f22d1e.png)

### Fold 3
![Screen Shot 2020-12-30 at 1 01 10 PM](https://user-images.githubusercontent.com/32279312/103381335-51a62d80-4aa0-11eb-9f38-319e9b6608af.png)

Reply:
Shoot... yeah that's probably just a random seed thing or something.... I'll mess with it some and see if I can get a better example. Maybe just keep the seed but switch the folds I'm comparing...